src/components/Choice/src/Choice.vue

```diff
@@ -47,12 +48,19 @@ export default {
 			default: 'single-select',
 			validator: (modeValue) => ['single-select', 'multi-select'].includes(modeValue),
 		},
+
+		selectedColor: {
+			type: String,
+			default: '#222',
+			validator: (color) => chroma.valid(color),
+		},
 	},
 
 	data() {
 		return {
 			currentValue: this.selected,
 			isMultiSelect: this.mode === 'multi-select',
+			color: () => this.selectedColor,
```

Comment:
Not sure why you need a function here
```suggestion
			color: this.selectedColor,
```

Reply:
@mattmewton I have modified this so that all computed properties related to selected options have been moved to `getSelectedStyle()` to clarify that this is a function. This was done to make these provided properties reactive.